projects/dashboards/ecometrics/src/app/app.component.html

```diff
@@ -0,0 +1,5 @@
+<h1 class="mat-h1">
+    <img src="assets/geoengine.svg" class="logo" />
+    <div>ECOMETRICS</div>
+</h1>
+<app-dashboard></app-dashboard>
```

Comment:
Maybe add a loading indicator on the bottom of the app like in the EBV analyzer. Loading something on the map takes a sec.

Reply:
as the app, in contrast to the ebv analyzer, is scrollable, I put the progress bar below the map instead

fe0c59c5a3a691959227d02ba805231bc07349d8